analytics/src/analytics/cli.py

```diff
@@ -149,6 +149,30 @@ def test_connection() -> None:
     result.close()
 
 
+@export_app.command(name="db_export")
+def export_json_to_database(
+    # dataset: BaseDataset, # got RuntimeError: Type not yet supported with this defined here
+    sprint_file: Annotated[str, SPRINT_FILE_ARG],
+    issue_file: Annotated[str, ISSUE_FILE_ARG],
+) -> None:
+    """Export JSON data to the database."""
+    # Get the database engine and establish a connection
+    engine = db.get_db()
+    connection = engine.connect()
+
+    # get data and load from JSON
+    task_data = DeliverableTasks.load_from_json_files(
+        sprint_file=sprint_file,
+        issue_file=issue_file,
+    )
+
+    BaseDataset.to_sql(
+        output_table=task_data,
```

Comment:
The `output_table` argument is a string for the table name. Can you recheck this locally? Did you have a working version here?

Reply:
@acouch it worked in the sense that the command ran locally, but I haven't been able to find the data in the database

Reply:
Try instantiating the class, then applying the .to_sql method :

```python

  deliverables = DeliverableTasks.load_from_json_files(
    sprint_file=sprint_file,
    issue_file=issue_file,
  )

  deliverables.to_sql(
    output_table="github_project_data",
    engine=connection,
    replace_table=True
  )
```